docs/userguides/dependencies.md

```diff
@@ -1,6 +1,6 @@
 # Dependencies
 
-Ape downloads and caches dependencies in the `.ape/packages/<name>/<version-id>` directory where `<name>` refers to the name of the dependency and `<version-id>` refers to the version or branch of the package.
+Ape downloads and caches dependencies in the `.ape/packages/projects/<name>/<version-id>` directory where `<name>` refers to the full-name of the dependency (e.g. `"OpenZeppelin/openzeppelin-contracts"`) and `<version-id>` refers to the version or branch of the package.
```

Comment:
Because there are now 3 elements to maintain for dependencies (3 sub-folders):

* `projects/` - the raw project files, needed for basic local project compiling where it pulls in these sources to a temporary `.cache` folder when compiling.
* `manifests/` - contain the on-goingly built manifests, similar to `__local__.json` works, except it is less of a big deal of these ever can compile. However, when you do compile dependencies, their contract types go here (just like in `.build/__local__.json`.
* `apis/` - these are the data fields you populate in your `dependencies:` key or feed into `ape pm install` commands. This is needed because it allows you to manage dependencies anywhere, like if navigate away from the folder containing their source `ape-config.yaml`. As I have been using this feature, i find it also to be a good place to retain functioning config overrides.

- [ ] document all this more